src/gpodder/model.py

```diff
@@ -514,8 +518,12 @@ def get_playback_url(self, config=None, allow_partial=False):
             return url + '.partial'
 
         if url is None or not os.path.exists(url):
-            # FIXME: may custom downloaders provide the real url ?
-            url = registry.download_url.resolve(config, self.url, self, allow_partial)
+            if config.player.videoplayer_youtube_dl_support and registry.custom_downloader.resolve(None, None, self) is not None:
```

Comment:
Please can you clarify what the issue is with `registry.download_url.resolve`?

Reply:
Note that the FIXME comes from the code in current master. 

As far as the question 'may custom downloaders provide the real url ?' goes, I think the answer in the case of youtube and similar is 'No'. Simple methods of obtaining a download URL from e.g. youtube (like in gPodders youtube.py) do not always seem to work, while youtube-dl does.

Reply:
Thanks for taking the time to look into it! I don't have this magic function readily available, unfortunately.